neo/rawio/plexon2rawio/plexon2rawio.py

```diff
@@ -354,16 +383,12 @@ def _segment_t_stop(self, block_index, seg_index):
         end_time = (
             self.pl2reader.pl2_file_info.m_StartRecordingTime + self.pl2reader.pl2_file_info.m_DurationOfRecording
         )
-        return end_time / self.pl2reader.pl2_file_info.m_TimestampFrequency
+        return float(end_time / self.pl2reader.pl2_file_info.m_TimestampFrequency)
```

Comment:
I think we need to asure that the output is a float, the division is internal. I think that division between numpy float scalars is OK.

Reply:
Okay cool. I just wasn't sure since the last problem was between an int - numpy.scalar so I wasn't sure if float/numpy.scalar could also cause a problem or not.